src/Traits/HasSharedLogic.php

```diff
@@ -29,6 +29,20 @@ public function to($chatId): self
         return $this;
     }
 
+    /**
+     * Add a caption.
+     *
+     * @param string $caption
+     *
+     * @return $this
+     */
+    public function caption($caption): self
```

Comment:
I think caption can go into the Audio class itself since there are no captions with the standard message notification.

Reply:
I've putted `caption` here because it exists in other methods like `sendDocument` `sendPhoto` `sendVideo` and a lot more that we will maybe need later on!

Reply:
So I think we should split into two traits so it can be a generic one with very common logic and one for file type notifications. What do you think?

If someone adds a caption to the message notification and tries to send it, wouldn't it fail? Besides the confusing part.

Reply:
Yes, I'm with splitting traits. But what we gonna call the trait that contain `caption` to use it with the methods I told you before. What do you think about `HasCaption`!! 

If someone use `caption` with message notification it just skip it without failing.

Reply:
Yes, `HasCaption` is good.

Also, good to know it doesn't fail but let's not confuse people :)